public class TestException{
    public int inc(){
        int x = 1;
        try{
            x = 1.1;
            return x;
        } catch (Exception e){
            x = 2;
            return x;
        } finally {
            x = 3;
        }
    }
    public static void main(String[] args){
        TestException testException = new TestException();
        System.out.println(testException.inc());
    }
}